MAUI/PDF-Viewer/Custom-Bookmark.md

```diff
@@ -0,0 +1,138 @@
+
+{% tabs %}
+{% highlight c# %}
+
+var customBookmarks = pdfViewer.CustomBookmarks;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Add , Edit and Remove custom bookmark
+
+To add, edit, or remove custom bookmarks in a PDF viewer, you can use the `CustomBookmarks` collection. This property provides access to the collection of custom bookmarks that can be manipulated to modify the bookmarks displayed in the viewer's bookmark view.
+
+### Add custom bookmarks using the bookmark pane
+
+Custom bookmarks can be added using the floating button in the bookmark pane.
+
+![Custom bookmark floating button](Images\custom-bookmark.png)
+
+### Adding a Custom Bookmark programmatically
+
+To add a custom bookmark, you can create a new instance of Bookmark and add it to the `CustomBookmarks` collection:
+
+{% tabs %}
+{% highlight c# %}
+
+Bookmark newBookmark = new Bookmark()
+{
+    Name = "New Bookmark",
+    PageNumber = 1, // Page number where the bookmark should to navigate
+};
+
+pdfViewer.CustomBookmarks.Add(newBookmark);
+
+
+{% endhighlight %}
+{% endtabs %}
+
+### Rename custom bookmarks using the bookmark context menu
+
+Tap the context menu button on the custom bookmark to be edited and choose Rename and enter the desired name.
+
+![Custom bookmark context menu](Images\custom-bookmark-contextmenu.png)
+
+### Rename a Custom Bookmark programmatically
+
+To rename an existing custom bookmark, you can retrieve the bookmark from the `CustomBookmarks` collection and modify its properties:
+
+{% tabs %}
+{% highlight c# %}
+
+Bookmark bookmarkToEdit = pdfViewer.CustomBookmarks.FirstOrDefault(b => b.Title == "Chapter 2");
+if (bookmarkToEdit != null)
+{
+    bookmarkToEdit.Name = "Edited Bookmark Title";
+}
+
+
+{% endhighlight %}
+{% endtabs %}
+
+### Remove custom bookmarks using the bookmark context menu
+
+Tap the context menu button on the custom bookmark to be removed and tap `Delete`.
+
+### Removing a Custom Bookmark programmatically
+
+To remove a custom bookmark, you can remove it directly from the `CustomBookmarks` collection:
+
+{% tabs %}
+{% highlight c# %}
+
+Bookmark bookmarkToRemove = pdfViewer.CustomBookmarks.FirstOrDefault(b => b.Title == "Chapter 2");
+if (bookmarkToRemove != null)
+{
+    pdfViewer.CustomBookmarks.Remove(bookmarkToRemove);
+}
+
+
+{% endhighlight %}
+{% endtabs %}
+
+## Navigating to custom bookmark
+
+### Navigating using UI
+
+As mentioned above, you can show the outline view by setting the `IsOultineViewVisible` property to `true`. When the outline view is showing, you can tap on any bookmark to navigate to the destination pointed to that bookmark. 
+
+![Custom bookmark in .NET MAUI PDF Viewer](Images\custom-bookmark.png)
+
+### Navigating programmatically
+
+The PDF viewer allows the users to navigate to an custom bookmark using the `GoToBookmark` method. The below code snippet illustrates the same.
```

Comment:
The PDF viewer allows the users to navigate to an custom bookmark using the `GoToBookmark` method. The following code sample illustrates the same.